package.json

```diff
@@ -7,12 +7,14 @@
     "clean": "rimraf dist",
     "dev": "npm-run-all --parallel clean build migrate",
     "dev:start": "npm run dev && npm run start",
+    "dev:start:watch": "nodemon --watch src --exec 'npm run dev:start'",
     "start": "node dist/server.js",
     "build": "babel ./src -d dist",
     "migrate": "sequelize db:migrate",
     "migrate:test": "sequelize db:migrate:undo:all && npm run migrate",
     "postinstall": "npm run dev",
     "test": "export NODE_ENV=test && npm run migrate:test && nyc mocha --recursive --require babel-register --exit",
+    "test:watch": "nodemon --watch test --exec npm test",
```

Comment:
Thanks, a ton, for adding this (and `dev:start:watch`) in! 🙏

Reply:
Thanks for taking the time to review. Much appreciated 👍